Extensions/PlayerControl_Ext.cs

```diff
@@ -12,7 +12,7 @@ public class PlayerControl_Ext : MonoBehaviour
     public int pdllMenuCursorPos = 0;
 		// This is a dead variable...with a cute name!
     public bool meowBenjee = false;
-
+		public static int speed = 5;
```

Comment:
Okay I am a bit confused here: this line appears in your other pr too and I am wondering should it not belong here or does it belong in the other PR? talking about #9